src/components/List/Separator/Separators.scss

```diff
@@ -0,0 +1,15 @@
+.Divider {
+    width: 240px;
```

Comment:
shouldn't it be just 100%?

Reply:
In figma it has a set width, I'll ask Dima